### ISSUE TITLE
Bugs on https://woopread.com/ website

### ISSUE THREAD

**Issue by nc4N7w4D** [dteviot/WebToEpub#453]
Something is weird with the woopread website, where inserting it into the page shows this http://puu.sh/H401X.png, but opening it directly from the page has it working correctly http://puu.sh/H402s.png

**Comment by dteviot** [comment]
@nc4N7w4D
What do you mean by "inserting it into the page"?
Please give me a list of the steps you do, so I can reproduce.

**Comment by nc4N7w4D** [comment]
Sorry, so when I open the extension, and copy and paste the link, the extension shows the first screenshot, that's what I meant by insert. 

When I'm on the woopread website and then open the extension, it loads the actual chapters. 
Now I'm unsure if I used it wrong, or something is not working as intended.

**Comment by dteviot** [comment]
@nc4N7w4D
OK, I think what you're doing is:
1. Open WebToEpub in browser (with no other windows open)
2. Paste a URL into the "Starting URL" text box
3. Click "Load and Analyze"
4. And this doesn't work right.

Can you confirm?

Not surprising.  When you do that, WebToEpub just grabs that single HTML page, which doesn't have very much.
When you open the page in a browser, it loads a lot more into the tab.  (And WebToEpub reads everything in the tab.)

Basically, congratulations, you've discovered an ancient mode (that obviously doesn't work very well anymore.)
I should probably remove it.

WebToEpub works best if you.
1. In Browser, go to page with "table of contents" for the URLs to fetch
2. Click on the WebToEpub icon.

**Comment by nc4N7w4D** [comment]
I do confirm this was what I was doing. It worked on many other websites in that way, like https://www.wuxiaworld.com/, http://royalroadweed.blogspot.com/2017/01/moonlight-sculptor-table-of-content.html. But yes, thanks for that! Answered a curious question <3

**Comment by dteviot** [comment]
@nc4N7w4D
It depends on the site.
Some sites are simple, and everything WebToEpub needs on the initial web page.
However, other sites run JavaScript and fetch additional information.  e.g. When you click on the "Show More" tags.

**Comment by nc4N7w4D** [comment]
I'm going to be a pain and ask...would it be possible to run the extension straight on novelupdates.com? 

Due to the nature of many groups dropping and picking up the same thing, sometimes with very few chapters, it'd be very easy to click on the "Chapter Listing" button in novelupdates.com and parse the links directly through the website in order to create a singular epub.

**Comment by dteviot** [comment]
@nc4N7w4D 
>would it be possible to run the extension straight on novelupdates.com?

Sorry, no.  The owner of NovelUpdates has explicitly told me not to do that.
